Business resume homepage/resume.css

```diff
@@ -0,0 +1,163 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap');
```

Comment:
When you import a font like this, it's interesting to think about what the browser must do before it renders everything and how to manage that process. This rule is something called "render blocking". Look it up!

Reply:
Can you share a link about this, please?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Performance/Critical_rendering_path